src/SamplesApp/SamplesApp.UITests/Extensions/RectExtensions.cs

```diff
@@ -13,5 +14,10 @@ public static float GetBottom(this IAppRect rect)
 		{
 			return rect.Y + rect.Height;
 		}
+
+		public static Rectangle ToRectangle(this IAppRect rect)
+		{
+			return new Rectangle((int)rect.X, (int)rect.Y, (int)rect.Width, (int)rect.Height);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'rect' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5170992)